Libraries/LibWeb/SVG/SVGPatternElement.idl

```diff
@@ -0,0 +1,18 @@
+#import <SVG/SVGElement.idl>
+#import <SVG/SVGFitToViewBox.idl>
+#import <SVG/SVGURIReference.idl>
+
+// https://svgwg.org/svg2-draft/pservers.html#InterfaceSVGPatternElement
+[Exposed=Window]
+interface SVGPatternElement : SVGElement {
+    [SameObject] readonly attribute SVGAnimatedLength x;
+    [SameObject] readonly attribute SVGAnimatedLength y;
+    [SameObject] readonly attribute SVGAnimatedLength width;
+    [SameObject] readonly attribute SVGAnimatedLength height;
+    [FIXME, SameObject] readonly attribute SVGAnimatedEnumeration patternUnits;
+    [FIXME, SameObject] readonly attribute SVGAnimatedEnumeration patternContentUnits;
+    [FIXME, SameObject] readonly attribute SVGAnimatedTransformList patternTransform;
```

Comment:
Please keep this in the order it appears in the spec, that just makes it easier to find things and see if the spec has changed.